src/cli.mjs

```diff
@@ -42,15 +50,41 @@ export async function run(_argvInput, _container) {
     let subMessage = chalk.reset(
       "Use 'fauna <command> --help' for more information about a command.",
     );
+    let epilogue = "";
 
     if (argvInput.length > 0) {
-      subMessage = chalk.red(e.message);
+      // If the error isn't one of our known errors, wrap it in a generic error message.
+      if (isUnknownError(e)) {
+        subMessage = chalk.red(
+          `An unexpected error occurred...\n\n${e.message}`,
+        );
+        epilogue = `\n${BUG_REPORT_MESSAGE}`;
+
+        logger.debug(`unknown error thrown: ${e.name}`);
+      } else {
+        // Otherwise, just use the error message
+        subMessage = chalk.red(e.message);
+      }
     }
-    const message = `${chalk.reset(await builtYargs.getHelp())}\n\n${subMessage}`;
-    logger.stderr(message);
+
+    // If the error has a truthy displayHelp property, render the help text. Otherwise, just use the error message.
```

Comment:
if not falsy hideHelp instead?

Reply:
or this?
```js
if (!e.hideHelp){
  logger.stderr(`${chalk.reset(await builtYargs.getHelp())}\n\n`}${subMessage}`);
}
```

Reply:
Ah, yeah, I had it inverted but that broke a lot of the current assumptions. I'll update.